recipes/pybacktrack/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pybacktrack" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6277fc5263e2a38c1c472435fea612fa9fb7923a14ba33b4dc30ba2e984411e7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pytest-runner
+    - pip
+  run:
+    - python >=3.5
+    - gmt >=5.0.0
+    - numpy
+    - pygplates
+    - scipy
+
+test:
+  imports:
+    - pybacktrack
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/EarthByte/pyBacktrack
+  summary: A tool for reconstructing paleobathymetry on oceanic and continental crust.
+  dev_url: https://github.com/EarthByte/pyBacktrack
+  doc_url: https://pybacktrack.readthedocs.io
+  license: GPL-2.0-or-later
```

Looks like it's actually
```suggestion
  license: GPL-2.0-only
```

---

Ah thanks. I realize now that it's the license header in the source files that determines this (rather than the LICENSE file).

---

Yeah, it took me bit too.